dds_web/forms.py

```diff
@@ -62,6 +66,20 @@ class RegistrationForm(flask_wtf.FlaskForm):
     confirm = wtforms.PasswordField("Repeat password")
     submit = wtforms.SubmitField("submit")
 
+    def validate_username(self, username):
+        user = models.User.query.filter_by(username=username.data).first()
+        if user:
+            raise wtforms.validators.ValidationError(
+                "That username is taken. Please choose a different one."
+            )
```

Comment:
There's some characters not allowed in usernames, for example `.`, should we check that already in the WTForms? I believe that would give a better user experience since otherwise it would fail on user creation?

Reply:
Yes, I will add that check, thanks.